truss-chains/examples/whisper/whisper.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional
+
+# flake8: noqa F402
+# This location assumes `fde`-repo is checked out at the same level as `truss`-repo.
+_LOCAL_WHISPER_LIB = "../../../../fde/whisper-trt/src"
+import sys
+
+sys.path.append(_LOCAL_WHISPER_LIB)
+
```

Comment:
this sys only needs to happen in local, not in the deployed chainlet. I wonder if this is the best way to write it.

Reply:
In remote this is just a no-op, since this path will lead to nothing... do you have strong feelings about leaving it here?

What would you like to see instead? Something like a guard block?

```
if chains.is_local:
  ...
```

?

Reply:
no strong feelings.